testlibrary/src/main/java/com/example/testlibrary/LibraryActivity.java

```diff
@@ -1,18 +1,28 @@
 package com.example.testlibrary;
 
+import android.app.Activity;
+import android.content.Intent;
 import android.os.Bundle;
 import android.widget.Button;
 
 import androidx.appcompat.app.AppCompatActivity;
 
 public class LibraryActivity extends AppCompatActivity {
 
+    public static String EXTRA_MESSAGE = "com.example.testlibrary.EXTRA_MESSAGE";
```

Comment:
`MainActivity` can use this key to extract the return message from the activity result's `Intent`.